README.md

```diff
@@ -120,6 +123,131 @@ and it should work.
 
 These rules are not supported by Google's Go team.
 
+<a name="go_repositories"></a>
+## go\_repositories
+
+```bzl
+go_repositories()
+```
+
+Instantiates external dependencies to Go toolchain in a WORKSPACE.
+All the other workspace rules and build rules assume that this rule is
+placed in the WORKSPACE.
+
+
+<a name="go_repository"></a>
+## go\_repository
+
+```bzl
+go_repository(name, importpath, commit, tag)
+```
+
+Fetches a remote repository of a Go project. It is an analogy to
+`git_repository` but it recognizes importpath redirection of Go.
```

Comment:
`git_repository` also expects `BUILD` files to be present. So there's no difference.
But I have explicitly mentioned the expectation.